### ISSUE TITLE
Enhanced information in the CI build logs

### ISSUE THREAD

**Issue by matthiasrichter** [alisw/ali-bot#417]
Some nice-to-have features for the ali-bot for the CI connected to PRs

* add time stamp for start and end of build to the log
* add information about the aliBuild target, options and defaults
* Github integration of ali-bot: link the items in the summary of CI checks to the log on the build server

**Comment by ktf** [comment]
Let's rediscuss this once @TimoWilken is done with his rework of the logs.
